modules/@apostrophecms/admin-bar/index.js

```diff
@@ -100,17 +103,6 @@ module.exports = {
         self.items.push(item);
       },
 
-      // Group several menu items together in the interface (currently
-      // implemented as a dropdown menu). If `items` is an array of menu
-      // item names, then the group's label is the same as the label of
-      // the first item. If you wish the label to differ from the label
-      // of the first item, instead pass an object with a `label` property
-      // and an `items` property.
-
-      group(items) {
```

Comment:
I could not find this used. Items group in the menu and tests pass without it.

Reply:
The workflow module uses it to programmatically create a reasonable grouping of its options. Without this there isn't a good way to do that in a module that doesn't "own" top level configuration.

Reply:
It's possible this should be left in place for now, but a beta ticket opened to think more about merging admin bar configuration versus the current "if the dev sets it up workflow's suggestions are ignored anyway" situation.

Reply:
Gotcha. I'll create a ticket in the workflow epic related to this.